app/models/dish.rb

```diff
@@ -1,4 +1,12 @@
 class Dish < ApplicationRecord
   validates_presence_of :name, :description
   belongs_to :chef
+  has_many :dish_ingredients
+  has_many :ingredients, through: :dish_ingredients
+
+  def all_ingredient_names
+    ingredients.map do |ingredient|
+      ingredient.name
+    end
```

Comment:
This is using Ruby which is not as efficient as using AR. How would you rewrite this using just AR?